samples/kubernetes/end2end/docs/01-pv-path.md

```diff
@@ -1,10 +1,10 @@
 ## Configuring the PV Path
-In this sample we need to create three Persistent Volumes (PV) and Persistent Volume Claims (PVC) to store data for MySQL, Prometheus, and Grafana. See the YAML files for the three PVs/PVCs.
+In this sample, we need to create three Persistent Volumes (PV) and Persistent Volume Claims (PVC) to store data for MySQL, Prometheus, and Grafana. See the YAML files for the three PVs/PVCs.
 - [PV and PVC for MYSQL server](../mysql/persistence.yaml).
 - [PV and PVC for Prometheus server](../prometheus/persistence.yaml).
 - [PV and PVC for Grafana server](../grafana/persistence.yaml).
 
-> **Note**: To simply the configuration, we use `hostPath` PV that can only work in a single-node Kubernetes cluster so it's for demonstration purposes only. In a production environment, you would need to use a more sophisticated PV type like NFS, iSCSI. For more details, see the Kubernetes [Types of Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes) document.  
+> **Note**: To simply the configuration, we use `hostPath` PV that can only work in a single-node Kubernetes cluster, therefore, it's for demonstration purposes only. In a production environment, you would need to use a more sophisticated PV type like NFS, iSCSI. For more details, see the Kubernetes [Types of Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes) document.  
```

Comment:
shouldn't that be a semicolon before "therefore"?

Reply:
YES. Thank you.

Reply:
Yes.